cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -593,17 +589,22 @@ func (c *CStorPoolController) reconcileVersion(csp *apis.CStorPool) (*apis.CStor
 				return csp, err
 			}
 		}
-		path := cspObj.VersionDetails.Path()
+		path := strings.Split(cspObj.VersionDetails.Status.Current, "-")[0]
```

Comment:
Can we return `strings.Split(cspObj.VersionDetails.Status.Current, "-")[0]` inside Path() func

Reply:
Currently its a one liner function. If it becomes something bigger will move it.